external/anomaly/tests/anomaly_common.py

```diff
@@ -107,3 +107,19 @@ def get_anomaly_domain_test_action_classes(
         OTETestNNCFExportEvaluationAction,
         OTETestNNCFGraphAction,
     ]
+
+
+def get_dummy_compressed_model(task):
```

Comment:
Would it be possible to annotate the args with type?

Reply:
done